docs/Product-Categories/WisLink/RAK2247/Quickstart/README.md

```diff
@@ -133,6 +156,28 @@ sudo ./lora_pkt_fwd
  If you added the additional lines in step 5 it will execute every time on boot.
 :::
 
-8. The regional parameter configurations for all the supported regions are located in the folder `</opt/packet_forwarder/lora_pkt_fwd/global_conf>`. In case you need to adjust the region frequency band for example, do so before running the process (**EU868 is the default**)
-
-**Congratulations!** :tada: you should now see your Gateway in TTN!
\ No newline at end of file
+10. To check if it is working, run `sudo systemctl start ttn-gateway.service` to start the service and check its status `service ttn-gateway status`. You should see something like the box bellow. Be aware that it may take some minutes to see your gateway as connected in TTN's console.
+```sh
+pi@raspberrypi:~ $ service ttn-gateway status
+● ttn-gateway.service - The Things Network Gateway
+   Loaded: loaded (/lib/systemd/system/ttn-gateway.service; disabled; vendor preset: enabled)
+   Active: active (running) since Fri 2020-10-09 17:50:55 BST; 1min 9s ago
+ Main PID: 721 (start.sh)
+    Tasks: 6 (limit: 2065)
+   CGroup: /system.slice/ttn-gateway.service
+           ├─721 /bin/bash /opt/ttn-gateway/packet_forwarder/lora_pkt_fwd/start.sh
+           └─769 ./lora_pkt_fwd
+
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: src/jitqueue.c:448:jit_print_queue(): INFO: [jit] queue is empty
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ### [GPS] ###
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: # GPS sync is disabled
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ##### END #####
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: JSON up: {"stat":{"time":"2020-10-09 16:51:30 GMT","rxnb":0,"rxok":0,"rxfw":0,"ackr":0.0,"dwnb":0,"t
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: INFO: [down] PULL_ACK received in 368 ms
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ##### 2020-10-09 16:52:00 GMT #####
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: ### [UPSTREAM] ###
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: # RF packets received by concentrator: 0
+Oct 09 17:52:00 raspberrypi ttn-gateway[721]: # CRC_OK: 0.00%, CRC_FAIL: 0.00%, NO_CRC: 0.00%
+
```

Comment:
Close the code block with ```

Reply:
Just did the spelling corrections and closed the code block. Thank you ;p

Reply:
BTW, I totally forgot to use the _spelling checker tool_ when I wrote this 🤦